module/Company/view/partial/company/admin/editors/label.phtml

```diff
@@ -0,0 +1,43 @@
+<div class="row">
+    <?php foreach( $form->getLanguages() as $lang):?>
+    <?php $langDesc = $languages[$lang]?>
+        <div class="col-md-6">
+            <h1><?= $this->translate($langDesc)?></h1>
+            <?php $fieldset = $this->form->get("labels")->get($lang); ?>
+            <?php
+                $element = $fieldset->get('name');
+                $element->setAttribute('class', 'form-control');
+            ?>
+            <div class="form-group<?= $this->bootstrapElementError($element) ?>">
+                <label for="<?= $element->getName() ?>" class="control-label col-sm-4">
```

Comment:
Unlike the old `count` method for the Bootstrap errors, `BootstrapElementError` does not have a space before the `has-error`. Thus there must be a space between `form-group` and `<?=`.

This issue appears 3 times in this document.